regulations/static/regulations/js/source/views/drawer/drawer-tabs-view.js

```diff
@@ -17,6 +17,7 @@ var DrawerTabsView = Backbone.View.extend({
 
     idMap: {
         'table-of-contents': '#menu-link',
+        'table-of-contents-secondary': '#table-of-contents-secondary-link',
         'timeline': '#timeline-link',
         'search': '#search-link'
```

Comment:
We can delay this for later, but can this be generalized by naming these more consistently? Just swapping `#menu-link` for `#table-of-contents-link`?